### DISCUSSION TITLE
Cannot delete attributes with no products

### DISCUSSION THREAD

**Discussion by haf** [swellstores/community#98] [Help] [answered]
<img width="372" alt="Screenshot 2023-01-22 at 15 50 12" src="https://user-images.githubusercontent.com/193115/213922228-167054f1-5361-46f5-aace-ff97e8761bcf.png">

What the title says.

API says 200 OK, but doesn't delete it.

<img width="440" alt="image" src="https://user-images.githubusercontent.com/193115/213922559-84685b58-3ce7-4abf-8eb9-3f092b348629.png">

**Comment by ericingram** [comment]
@haf Share your account name with support@swell.is so we can look into this. I'd guess it's an issue with an archived variant. If so, a workaround would be to delete any archived variant using that attribute.

**Comment by logeshswell** [comment, marked as answer]
@haf This seems like a frontend bug and I will raise a bug ticket internally for faster resolution.
As a temporary workaround, I would suggest you follow the below instructions to delete the attribute.
1. Click **Actions** > `View in console`.
2. Click `Edit` hyperlink and then **Delete** tab.
3. Click **Run** button to delete the attribute.
